urwid/widget/widget.py

```diff
@@ -284,58 +284,6 @@ class Widget(metaclass=WidgetMeta):
        widget renders the same size regardless of the value of the *focus*
        parameter.
 
-
```

Comment:
These parts of docstring duplicates implemented methods and causes warnings during documentation build.